lib/internal/Magento/Framework/RequireJs/Config.php

```diff
@@ -293,24 +293,26 @@ protected function getConfigFileName()
      */
     public function getMinResolverCode()
     {
-        $excludes = ['url.indexOf(baseUrl) === 0'];
+        $excludes = ['url.indexOf(baseUrl)===0'];
```

Comment:
Why did you remove spaces here?

Reply:
This is for consistency as the other parts of the condition added by https://github.com/magento/magento2/pull/27622/files#diff-c0df1f93853550f5aa5d365ef36c307aR298 don't have spaces between them.